ctdcal/convert.py

```diff
@@ -51,6 +53,62 @@ def convertFromFiles(hex_file, xmlcon_file, debug=False):
     return convertFromSBEReader(sbeReader, DEBUG)
 
 
+def hex_to_ctd(ssscc_list, debug=False):
+    # TODO: add (some) error handling from odf_convert_sbe.py
+    """
+    Convert raw CTD data and export to .pkl files.
+
+    Parameters
+    ----------
+    ssscc_list : list of str
+        List of stations to convert
+    debug : bool, optional
+        Display verbose messages
+
+    Returns
+    -------
+
+    """
+    print('Converting .hex files')
```

Comment:
Any plans in the works to convert these to use a [logger](https://docs.python.org/3.8/library/logging.html) rather than print?

Reply:
eventually yes, not very high priority at the moment

Reply:
added todo item for implementing logger